ecms_base/modules/custom/ecms_layout/src/ECMSLayout.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\ECMS_layout;
```

Comment:
The namespace should be `Drupal\ecms_layout` to match the other classes below. I'd also rename the file to `EcmsLayout.php` and the class to `EcmsLayout`.